TESTS/mbed_hal/trng/base64b/base64b.h

```diff
@@ -0,0 +1,16 @@
+#include <stdint.h>
```

Comment:
license header missing in some files (please review), not only this one

Reply:
Sorry for that, all license header will be added.